tests/phpunit/tests/test-install.php

```diff
@@ -16,10 +16,29 @@ function test_activate() {
 		$this->assertEquals( POLYLANG_VERSION, $options['version'] );
 	}
 
+	/**
+	 * This test requires the definition of the constants WP_UNINSTALL_PLUGIN
+	 * The constant PLL_REMOVE_ALL_DATA must not be defined
+	 */
 	function test_no_uninstall() {
-		$this->assertFalse( defined( 'PLL_REMOVE_ALL_DATA' ), 'The constant PLL_REMOVE_ALL_DATA mut not be defined for this test'  );
+		if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
```

Comment:
This still bothers me that we don't know whether this constant is defined or not in our environment. In fact, it might even be better to define it anyway, so if the test outputs an error, it will give us an insight on our test environment.

Reply:
Do you mean taht you would prefer to define the constant `WP_UNINSTALL_PLUGIN` in the boostrap once and for all?

Reply:
On second thought, no, that would be less explicit. Just let it as it is.